node/src/rpc.rs

```diff
@@ -94,14 +94,10 @@ pub struct FullDeps<C, P, SC, B> {
 	pub babe: BabeDeps,
 	/// GRANDPA specific dependencies.
 	pub grandpa: GrandpaDeps<B>,
-	/// The maximum number of cells that can be requested in one go.
-	pub kate_max_cells_size: usize,
-	/// Enable Kate RPCs
-	pub kate_rpc_enabled: bool,
-	/// Enable Kate RPCs Metrics
-	///
-	/// Should not be used unless unless you know what you're doing.
-	pub kate_rpc_metrics_enabled: bool,
+	/// TODO
```

Comment:
Can we resolve this todo and just mention the configs which are wrapped inside it?

Reply:
Done. Thank you